### PR TITLE
update README files and related community files

### LINKED ISSUE #2: update README file and linked community files
- [x] CONTRIBUTING.md
- [x] LICENSE.md
- [x] CODE_OF_CONDUCT.md
- [x] SUPPORT.md
- [x] README.md

### PR DESCRIPTION
**Pull request by avallecam**
Fixes #2

### CODE REVIEW

**Comment by avallecam** [comment]
@Bisaloo, the changes here are entirely based on the `tutorials` repo. 

So, the new content in which I request your specific review is in two sections where I refer to the original repository:
- top line in the [LICENSE file](https://github.com/epiverse-trace/git-rstudio-basics/pull/3/files#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1dR7),
- Usage section in the [README file](https://github.com/epiverse-trace/git-rstudio-basics/pull/3/files#diff-72778b58969c8ca8268402860b0e003e3d213a26c812bc9f9b928395c284c99fR26-R28).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +10,7 @@ welcome.
 By contributing, you agree that we may redistribute your work under [our
 license](LICENSE.md). In exchange, we will address your issues and/or assess
 your change proposal as promptly as we can, and help you become a member of our
-community. Everyone involved in [The Carpentries][cp-site] agrees to abide by
+community. Everyone involved in [The Epiverse-TRACE][cp-site] agrees to abide by
```
Comment (Bisaloo) [inline CONTRIBUTING.md:13]:
```suggestion
community. Everyone involved in [Epiverse-TRACE][cp-site] agrees to abide by
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,26 +168,26 @@ final say over what gets merged into the lesson.
...
+teaching and learning. There are several ways to connect with The Epiverse-TRACE
+community listed at <https://github.com/epiverse-trace/> including via social
 media, slack, newsletters, and email lists. You can also [reach us by
 email][contact].
 
-[repo]: https://github.com/swcarpentry/git-novice
-[repo-issues]: https://github.com/swcarpentry/git-novice/issues
-[contact]: mailto:team@carpentries.org
-[cp-site]: https://carpentries.org/
+[repo]: https://example.com/FIXME
```
Comment (Bisaloo) [inline CONTRIBUTING.md:178]:
```suggestion
[repo]: https://github.com/epiverse-trace/git-rstudio-basics
```

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Start by making a minimal **repr**oducible **ex**ample using the  [reprex](https://reprex.tidyverse.org/) package. 
+If you haven't heard of or used reprex before, you're in for a treat! 
+Seriously, reprex will make all of your R-question-asking endeavors easier (which is a pretty insane ROI for the five to ten minutes it'll take you to learn what it's all about). 
+For additional reprex pointers, check out the [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site.
+You can also check out the [How to make a reproducible R code example](https://community.appliedepi.org/t/how-to-make-a-reproducible-r-code-example/167/1) of the AppliedEpi community site.
+
+## Where to ask?
+
+Armed with your reprex, the next step is to figure out [where to ask](https://www.tidyverse.org/help/#where-to-ask).
```
Comment (Bisaloo) [inline SUPPORT.md:16]: This link should probably be updated or removed.

Comment (avallecam) [reply SUPPORT.md]: Yep, I move it from the top to the last line about asking to the community https://github.com/epiverse-trace/git-rstudio-basics/pull/3/commits/bef3b35ee3297fd89eaf1b1a54560bb84d44eca4

**Comment by avallecam** [comment]
Thank you for the thorough review, @Bisaloo I accepted your suggestions and made a commit to your comment.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -70,9 +68,84 @@ second, to think hard about priorities.
...
+
+- The number `##` is equal to the PR number. 
+- `.md` files contain [generated lesson content](https://carpentries.github.io/sandpaper-docs/deployment.html) for review with knitted text, code, and figures. 
+- These `.md` files generate the `.html` of the website and look similar to how it's going to look like online.
+- Locally, after switching to the review branch in Rstudio, you can use the [Visual editor](https://posit.co/blog/exploring-rstudio-visual-markdown-editor/), or Preview it as an HTML file (which later you will need to delete).
+- Make any edit suggestions in the [PR online on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).
+
+#### Maintainer:
+
+You can follow this references:
```
Comment (Bisaloo) [inline CONTRIBUTING.md:125]:
```suggestion
You can follow this reference:
```

**Review by Bisaloo — APPROVED** [review]

**Pull request merged**